src/libCrypto/EthCrypto.cpp

```diff
@@ -334,3 +334,40 @@ bytes GetOriginalHash(TransactionCoreInfo const& info, uint64_t chainId) {
 
   return bytes{&signingHash.bytes[0], &signingHash.bytes[32]};
 }
+
+bytes ToEVM(bytes const& in) {
+  if (in.empty()) {
+    return in;
+  }
+
+  std::string prefixedEvm{"EVM"};
+  prefixedEvm += DataConversion::Uint8VecToHexStrRet(in);
+  bytes ret;
+
+  std::copy(prefixedEvm.begin(), prefixedEvm.end(), std::back_inserter(ret));
+
+  return ret;
+}
+
+bytes FromEVM(bytes const& in) {
+  if (in.empty()) {
+    return in;
+  }
+
+  // std::string ret =
+  // std::string(reinterpret_cast<const char*>(in.data()), in.size());
+  // ret.erase(0, 3);
+
+  // Copy all but the first three characters('EVM') into a string for conversion
+  // Advance iterator at beginning
+  std::string ret;
```

Comment:
Nitpick: `ret{in.begin() + 3, in.end()};`

Reply:
👍